01-introduction.md

```diff
@@ -74,11 +74,11 @@ so that they get the positive feedback of being able to read whole stories as ea
 The basic idea is that once children find reading rewarding,
 they'll be motivated to go back and figure out the letters.
 
-Both approaches are consistent with what we know about how children learn,
-so we need to study them in action to determine which is better.
-The findings have been contradictory:
-while some studies find that phonics works best,
-others show that whole language yields better results.
+These approaches are not equally consistent with what we know about how children
+learn. In fact, the scientific consensus today is that the whole word approach
+was a mistake. Nevertheless, when these methods were studied in classrooms, the
```

Comment:
On Sat, Jan 09, 2016 at 03:55:36PM -0800, Ariel Rokem wrote:
> > +These approaches are not equally consistent with what we know
> > about how children learn. In fact, the scientific consensus today
> > is that the whole word approach was a mistake. Nevertheless, when
> > these methods were studied in classrooms, the
> 
> I've asked my colleague for key references on the issue, and I hope
> he will respond with something better, but for the time being here's
> one survey of the literature on this:
> http://www.haskins.yale.edu/sr/SR111/SR111_13.pdf

This is way outside of my field, and I only skimmed the article, but
it seems to focus on whether “phonomic awareness”, not “reading
comprehension”.  It seems pretty obvious to me that focusing on
letters and phonemic segmentation would help with phonomic awareness
more than whole-word logographic reading.  The primary research laid
out in Table 1 does suggest that phonemic-segmentation training is
more effective in teaching reading (pronunciation?), but the authors
themselves throw in some caveats about Hebrew reading tests before
calling their results “suggestive”.  If I'm missing the bit that
suggests the field as a whole has decided that the whole-word approach
to teaching reading comprehension is a mistake, please point me at the
right part of this paper ;).